pytransform3d/test/test_rotations.py

```diff
@@ -1160,6 +1160,17 @@ def test_quaternion_from_matrix_180_not_axis_aligned():
         pr.assert_quaternion_equal(q, q_from_R)
 
 
+def test_quaternion_from_extrinsic_euler_xyz():
+    """Test quaternion_from_extrinsic_euler_xyz."""
+    random_state = np.random.RandomState(0)
+    for i in range(10):
+        e = random_state.uniform(-100, 100, [3])
```

Comment:
The recommended range is [-pi, pi] for alpha, [-pi/2, pi/2] for beta, and [-pi, pi] for gamma. (https://dfki-ric.github.io/pytransform3d/euler_angles.html#range-of-angles)

Reply:
Yes, I understand the recommended range can lead to a unique representation. But for the unit test, we might need to assume a larger range since users usually can not realize it.

Reply:
All right.